packages/wonder-blocks-modal/components/close-button.js

```diff
@@ -38,14 +24,17 @@ export default class CloseButton extends React.Component<Props> {
                             "You've specified 'onClose' on a modal when using ModalLauncher.  Please specify 'onClose' on the ModalLauncher instead",
                         );
                     }
+
                     return (
                         <IconButton
                             icon={icons.dismiss}
                             // TODO(mdr): Translate this string for i18n.
+                            // TODO(kevinb): provide a way to set this label
                             aria-label="Close modal"
-                            onClick={closeModal || onClick}
+                            onClick={onClick || closeModal}
```

Comment:
Why are these reversed?

Reply:
Not sure why I reversed them.  Only one is possible at a time since we throw an error if both are provided.